protected/views/layouts/new_datasetpage.php

```diff
@@ -6,6 +6,8 @@
     <meta name="language" content="en" />
     <meta name="robots" content="noindex">
     <meta name="googlebot" content="noindex">
```

Comment:
Hi @pli888 and @rija 

Change the `noindex` to `index` here should fix the problem mentioned in issue #515. 
Reference can be found at [Google Search Central](https://developers.google.com/search/docs/advanced/crawling/special-tags).

Reply:
>Change the noindex to index here should fix the problem mentioned in issue #515.

@kencho51 This still needs to be done.